README.md

```diff
@@ -1,11 +1,46 @@
 # SwapScope
 
-Welcome to **SwapScope**, a monorepo that houses the SwapScope project. This repository is composed of two distinct components: the **DApp** (Decentralized Application) and the **Publisher**. These components are organized into separate folders for your convenience.
+Swapscope is an open-source project that simplifies the process of streaming real-time Ethereum Uniswap (for now) liquidity operations (additions and removals) data. The project consists of two main components: the DApp (frontend) and the publisher (backend). 
+Swapscope's publisher utilizes [Syntropy Data Layer's (DL)](https://www.syntropynet.com/post/presenting-the-new-vision) Ethereum event log stream, processes the data, and publishes it back to Syntropy DL which frontend also utilizes to display the processed data.
 
-## Project Overview
+## Getting Started
 
-TBD
+### Prerequisites
+Before you start using (modifying) Swapscope locally, there are some prerequisites you need to fulfill:
+* Access to Syntropy Data Layer and it's streams - [Developer Portal](https://developer-portal.syntropynet.com/)
+* Go (Golang 1.20) - [Install Go](https://go.dev/doc/install)
+* Node.js and npm - [Install Node.js and npm](https://nodejs.org/en)
+* Access to Coingecko API (for token and pricing information (for now)) - [Coingecko API](https://www.coingecko.com/en/api)
+* Local database for storing token and liquidity pool information (helps to reduce API calls significantly) + publisher has built-in functionality to save liquidity addition/removal operations into a database
 
-## Getting Started
+### Usage
+* Latest release images can be found here: https://github.com/SyntropyNet/swapscope/releases/latest
+* DApp instructions: [DApp README.md](https://github.com/SyntropyNet/swapscope/blob/main/dapp/README.md)
+* Publisher instructions: [Publisher README.md](https://github.com/SyntropyNet/swapscope/blob/main/publisher/README.md)
+
+### Installation
+<span style="color:red"> *TODO: is there a command to run DApp and publisher at the same time?*</span>
```

Comment:
We could use docker-compose to start the database as well as dApp and Publisher containers.

Reply:
Database part covered here: https://github.com/SyntropyNet/swapscope/issues/16